Project/Source/Modules/ModuleRender.cpp

```diff
@@ -252,6 +258,18 @@ void ModuleRender::ViewportResized(int width, int height) {
 	viewportUpdated = true;
 }
 
+void ModuleRender::ReceiveEvent(TesseractEvent& e) {
+	switch (e.type) {
+	case TesseractEventType::SCREEN_RESIZED:
+#if GAME
+		ViewportResized(App->window->GetWidth(), App->window->GetHeight());
```

Comment:
Not too sure about this. Compare with ViewportResized and make sure every ViewportResized is called.

Reply:
Refactored, now using events.